DOL.WHD.Section14c.Web/src/v4/dol-footer.component.ts

```diff
@@ -24,30 +24,43 @@ import { Component } from '@angular/core';
             200 Constitution Ave. NW<br>
             Washington DC 20210<br>
-            1-866-4-USA-DOL<br>
-            1-866-487-2365<br>
-            <a href="https://www.dol.gov/">www.dol.gov</a>
+            <a href="tel:1-866-487-2365">1-866-4-USA-DOL</a><br>
+            <a href="tel:1-866-487-2365">1-866-487-2365</a><br>
+            <a href="http://www.dol.gov/dol/contact/contact-phonecallcenter.htm">TTY</a><br>
+            <a href="http://www.dol.gov/">www.dol.gov </a>
           </div>
-          <ul class="usa-unstyled-list usa-width-one-fourth content-group">
+          <ul class="usa-unstyled-list usa-width-one-fourth content-group" role="navigation">
             <li class="mb1"><strong>ABOUT THE SITE</strong></li>
-            <li><a href="#">Link 1</a></li>
-            <li><a href="#">Link 2</a></li>
-            <li><a href="#">Link 3</a></li>
-            <li><a href="#">Link 4</a></li>
+            <li><a href="http://www.dol.gov/dol/foia/">Freedom of Information Act</a></li>
+            <li><a href="http://www.dol.gov/dol/privacynotice.htm">Privacy &amp; Security Statement</a></li>
+            <li><a href="http://www.dol.gov/dol/disclaim.htm">Disclaimers</a></li>
+            <li><a href="http://www.dol.gov/dol/aboutdol/website-policies.htm">Important Web Site Notices</a></li>
+            <li><a href="http://www.dol.gov/dol/aboutdol/plug-ins.htm">Plug-ins Used by DOL</a></li>
+            <li><a href="http://www.dol.gov/rss/">RSS Feeds from DOL</a></li>
+            <li><a href="http://www.dol.gov/dol/aboutdol/accessibility.htm">Accessibility Statement</a></li>
           </ul>
-          <ul class="usa-unstyled-list usa-width-one-fourth content-group">
+          <ul class="usa-unstyled-list usa-width-one-fourth content-group" role="navigation">
             <li class="mb1"><strong>LABOR DEPARTMENT</strong></li>
-            <li><a href="#">Link 1</a></li>
-            <li><a href="#">Link 2</a></li>
-            <li><a href="#">Link 3</a></li>
-            <li><a href="#">Link 4</a></li>
+            <li><a href="http://www.dol.gov/dol/topic/Spanish-speakingTopic.htm">Español</a></li>
+            <li><a href="http://www.dol.gov/dol/siteindex.htm">A to Z Index</a></li>
+            <li><a href="http://www.dol.gov/dol/organization.htm">Agencies</a></li>
+            <li><a href="http://www.oig.dol.gov/">Office of Inspector General</a></li>
+            <li><a href="http://www.dol.gov/dol/contact/contact-phonekeypersonnel.htm">Leadership Team</a></li>
+            <li><a href="http://www.dol.gov/dol/contact/">Contact Us</a></li>
+            <li><a href="https://public.govdelivery.com/accounts/USDOL/subscriber/new?topic_id=USDOL_167" onclick="window.open('https://public.govdelivery.com/accounts/USDOL/subscriber/new?topic_id=USDOL_167&amp;pop=t', '', 'scrollbars=1,toolbar=0,menubar=0,resizable=1,width=800,height=420');return false;">Subscribe to the DOL Newsletter</a></li>
```

Comment:
The URL in `window.open()` inside the `onclick` handler should also have a literal ampersand instead of `&amp;`.

Reply:
Fixed and pushed to the branch again.